bench/package.json

```diff
@@ -1,15 +1,16 @@
 {
     "name": "parse5-benchmarks",
+    "private": "true",
     "type": "module",
     "version": "1.0.0",
     "description": "parse5 regression benchmarks",
     "author": "Ivan Nikulin <ifaaan@gmail.com>",
     "license": "MIT",
     "dependencies": {
         "benchmark": "^2.1.4",
-        "human-format": "^0.7.0",
+        "human-format": "^0.11.0",
         "@airbnb/node-memwatch": "^2.0.0",
-        "parse5": "*",
+        "parse5": "npm:parse5",
```

Comment:
Why?

Reply:
The `perf` benchmark compares the published module (from NPM) to the unreleased module. Because we are now using NPM workspaces, the original entry would have pulled in the unreleased module from the `packages` directory. By explicitly asking for the NPM module here, we work around that issue.